src/controllers/article.js

```diff
@@ -22,5 +23,53 @@ export default {
     const articleData = await Articles.createComment({ articleId, comment });
 
     return response.status(201).json({ status: 'success', data: articleData, message: 'Comment added succesfully' });
+  },
+
+  /**
+      * controller for adding article to bookmark
+      *
+      * @function
+      *
+      * @param {Object} request - express request object
+      * @param {Object} response - express response object
+      *
+      * @return {Object} - the created bookmark
+      */
+  createBookmark: async (request, response) => {
+    const { articleId } = request.params;
+    const { user: userId } = request.body;
+
+    const articleObject = await Articles.getExistingArticle(articleId);
+
+    const existingBookmark = await Bookmarks.getExistingBookmark(articleId, userId);
+
+    if (existingBookmark) throw new ApplicationError(409, 'bookmark already added');
+
+    const newBookmark = await articleObject.createBookmark({ userId });
+
+    return response.status(201).json({ data: newBookmark, message: 'bookmark added succesfully' });
```

Comment:
Please let this error start with a Capital letter. I noticed a lot of people using Sentence case and decided that it was better to use what most apps use.
```suggestion
    return response.status(201).json({ data: newBookmark, message: 'Bookmark added succesfully' });
```